bot/resources/tags/dunder-methods.md

```diff
@@ -0,0 +1,28 @@
+### Dunder methods
+
+Double-underscore methods, or "dunder" methods, are special methods defined in a class that are invoked implicitly. Like the name suggests, they are prefixed and suffixed with dunders. You've probably already seen some, such as the `__init__` dunder method, also known as the "constructor" of a class, which is implicitly invoked when you instantiate an instance of a class.
+
+When you create a new class, all the default dunder methods are inherited from its superclass (which is `object` if no superclass is specified). However, we can override them by redefining the methods within this new class. For example, the default `__init__` method from `object` doesn't take any arguments, so we almost always override that to fit our needs.
```

Comment:
Perhaps you could word the inheritance portion a little differently? All methods are inherited, dunders are just special because they come with defaults. The way this is phrased suggests that the inheritance is special. 

For the curious: You can use `dir(object)` to see what's available by default and thus almost universally available.

Reply:
yep, rephrased